Project-Bootcamp/views/character-view.js

```diff
@@ -0,0 +1,78 @@
+var React = require ('react');
+var _ = require ('lodash');
+var $ = require ('jquery');
+var Button = require ('../components/button');
+var Input = require ('../components/input');
+var List = require ('../components/list');
+var BlizzardAPI = require ('../querys/query');
+
+var CharacterView = React.createClass ({
+
+    getInitialState: function () {
+
```

Comment:
I think I'll just leave a general comment here, since I'm seeing it's a constant... :P
Please remove all line breaks after you open a ``function``.